pkg/fanal/analyzer/secret/secret.go

```diff
@@ -39,6 +39,7 @@ var (
 
 type ScannerOption struct {
 	ConfigPath string
+	Config     *secret.Config
 }
 
 // SecretAnalyzer is an analyzer for secrets
```

Comment:
I think we can just export `scanner` in `SecretAnalyzer`. You can initialize SecretAnalyzer with your scanner rather than calling `RegisterSecretAnalyzer`.